resources/code_of_conduct.Rmd

```diff
@@ -0,0 +1,123 @@
+
+The organizers may take action to redress anything designed to, or with the clear impact of, disrupting an event or electronic communication platform or making the environment hostile for any community member.
+
+We expect everyone in the DaSEH community to comply with the Code of Conduct when participating in events and online communication platforms.
+
+## Reporting
+
+If someone makes you or anyone else feel unsafe or unwelcome, please report it as soon as possible. You can make a report either anonymously or personally. All reports will be reviewed by the organizers and will be kept confidential.
+
+#### Electronically
+
+You can make an anonymous or non-anonymous report via the following link: https://forms.gle/WBymAnzfWYpfWk4b6. It is a free-form text box that will be forwarded to the organizers. If you are uncomfortable reporting to the organizers as a group, you can contact any individual instructor via email or a direct message on the Slack workspace (https://daseh.slack.com/). Please include screenshots/copies of all relevant electronic conversations whenever possible (you don’t need to compromise your anonymity!).
+
+We can’t follow up an anonymous report with you directly, but we will fully investigate it and take whatever action is necessary to prevent a recurrence.
+
+#### Personal Report (for any DaSEH events: in-person or virtual)
+
+You can make a personal report to any member of the instructors or organizers present at an event.
+
+When taking a personal report, we will ensure you are safe and cannot be overheard. We may involve other event staff to ensure your report is managed properly. Once safe, we’ll ask you to tell us about what happened. This can be upsetting, but we’ll handle it as respectfully as possible, and you can bring someone to support you. You won’t be asked to confront anyone, and we won’t tell anyone who you are.
+
+Our team will be happy to help you get the relevant support (e.g. help contacting hotel/venue security, local law enforcement, local support services, provide escorts, or otherwise assist you to feel safe for the duration of the event).
+
+We value your attendance and participation in our community.
+
+## Ombudsperson
+
+We understand that you may want a person unrelated to DaSEH overseeing your conflict resolution. To facilitate this, we encourage community members to reach out to our designated third party service (EthicsPoint) contracted by Fred Hutch. See more information [here](https://secure.ethicspoint.com/domain/media/en/gui/89211/index.html). They can assist community members by offering consultation, mediation, coaching and referrals to other resources.
+
+## Incident Procedure
+
+The organizers follow a standard procedure to deal with conflicts. This procedure is briefly summarized below. Incident procedures are based on the book “How to Respond to Code of Conduct Reports” by Valerie Aurora and Mary Gardiner.
+
+#### Report Acknowledgement
```

Comment:
do we want to use the american spelling (Acknowledgment)? this word always trips me up :P